bin/generate.js

```diff
@@ -390,6 +391,22 @@ Creating Web Components People Love.
       }
       isMainBranch = true;
 
+      process.stdout.write(`\nConfiguring Husky scripts`);
+      loadingLoop(() => isHuskyConfigured);
+      try {
+        await exec('npx husky-init', { cwd: params.dir });
+        log(chalk.green('\nnpx husky-init'));
```

Comment:
How about we just have the one feedback that says, `Husky successfully configured!`

Then remove [401](https://github.com/AlaskaAirlines/WC-Generator/pull/121/files#diff-167b2ae1fd970f1cc86e298f93d8a9fb2701fb3c7b1acb815b09837eab4d75abR400-R401) and [404](https://github.com/AlaskaAirlines/WC-Generator/pull/121/files#diff-167b2ae1fd970f1cc86e298f93d8a9fb2701fb3c7b1acb815b09837eab4d75abR403-R404).